site/gatsby-site/src/components/submissions/SubmissionForm.js

```diff
@@ -48,13 +48,10 @@ export const schema = yup.object().shape({
     .min(6, '*Title must have at least 6 characters')
     .max(500, "*Titles can't be longer than 500 characters")
     .required('*Title is required'),
-  description: yup.string().when(['_id', 'incident_id'], {
-    is: (_id, incident_id) => _id !== undefined && (incident_id == '' || incident_id === undefined),
-    then: yup
-      .string()
-      .min(3, 'Description must have at least 3 characters')
-      .max(500, "Description can't be longer than 500 characters"),
-  }),
+  description: yup
```

Comment:
What's this change doing 🤔 ?

Reply:
The description field is no longer required, so we don't need to check that there's no incident id to place a length restriction. Until I made this change, the validation messages were not showing up for the description field – not sure why.